_data/rgdeekshith/rgdeekshith.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Rallapalli Goutham Deekshith
+institution: Hyderabad Institute of Technology and Management
+profile_pic: rgdeekshith.jpg
```

Comment:
Incorrect file name, please do one of the following:
1. Change `rgdeekshith.jpg` to `rgdeekshith.png` in the markdown file.
2. Rename `rgdeekshith.png` to `rgdeekshith.jpg`.
I'd suggest you make the change in the markdown file!